src/server/api/campaign.js

```diff
@@ -3,6 +3,8 @@ import { Campaign, JobRequest, r } from '../models'
 
 export function buildCampaignQuery(queryParam, organizationId, campaignsFilter, addFromClause = true) {
   let query = queryParam
+  const resultSize = (campaignsFilter.listSize ? campaignsFilter.listSize : 0)
+  const pageSize = (campaignsFilter.pageSize ? campaignsFilter.pageSize : 0)
```

Comment:
let's default pagesize to something other than 0 -- maybe 10? 
If `pageSize` is `offset' -- then maybe rename to `pageOffset` -- 'pagesize' sounds like it would mean the same thing as listsize/resultsize